src/article.less

```diff
@@ -0,0 +1,26 @@
+article {
+
+	.article-title {
+		font-size: 3rem;
+	}
+
+	.article-meta {
+		font-size: 15px;
+		color: @muted;
+
+		a {
+			color: @muted-text;
+			background-image: none;
+
+			&:hover {
+				color: #000;
```

Comment:
This should be using a colour variable instead of the hard coded value

Reply:
thank for your feedback, I have changed color value to variable